src/connectors/search-box/connectSearchBox.js

```diff
@@ -150,7 +153,7 @@ export default function connectSearchBox(renderFn, unmountFn) {
       },
 
       getWidgetSearchParameters(searchParameters, { uiState }) {
-        return searchParameters.setQuery(uiState.query || '');
+        return searchParameters.setQuery(uiState.query);
```

Comment:
This part is not exactly true. It doesn't matter much at the moment because it will be updated once we do the modifications inside `getConfiguration` to "mount" a default value on the state that indicates that a widget "own" this part of the state.